.github/workflows/release.yml

```diff
@@ -0,0 +1,53 @@
+name: Release X-Ray Node SDK to Github and NPM
+on:
+  workflow_dispatch:
+    inputs:
+      version:
+        description: The version to tag the release with, e.g., 1.2.0, 1.2.1-alpha.1
+        required: true
+
+jobs:
+  publish_xray_node_sdk:
+    name: Test and publish X-Ray Node SDK to NPM registry and Github
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout Repository
+        uses: actions/checkout@v2
+        with:
+          fetch-depth: 0
+
+      - name: Setup Node
+        uses: actions/setup-node@v1
+        with:
+          node-version: '14.x'
+          registry-url: 'https://registry.npmjs.org'
+
+      - name: Cache NPM modules
+        uses: actions/cache@v2
+        with:
+          path: |
+            node_modules
+            */*/node_modules
+          key: ubuntu-latest-${{ hashFiles('**/package.json') }}
```

Comment:
Use runner.os since it's our normal pattern.

Reply:
Yes. makes sense!